### PR TITLE
Renaming Truffle plugin, plus some typo fixes

### PR DESCRIPTION
**Pull request by bmmpxf**

### CODE REVIEW

File: docs/source/main/building-security-tools-on-mythx.rst [hunk 1]
```diff
@@ -302,9 +302,9 @@ The output contains a list of issues with title, short description and long desc
 - The `meta` field contains meta information about the analysis run.
 
 
-We recommend that users submit both bytecode and source code to obtain a full analysis. If only the creation bytecode is given, and not the source code, MythX will return a result like the folllowing:
+We recommend that users submit both bytecode and source code to obtain a full analysis. If only the creation bytecode is given, and not the source code, MythX will return a result like the following:
 
-.. code-block:: json
+.. code-block:: console
```
[the remark this answers is not in the document]
Comment (bmmpxf) [reply docs/source/main/building-security-tools-on-mythx.rst]: @dmuhs That was my first thought, but Sphinx treated that as an error (`expected 1 argument, got 0`) The language parser in recent versions of Sphinx is a little too fussy for me, though I understand where it's coming from.

**Comment by bmmpxf** [comment]
@dmuhs So what are the next steps? Can I merge to `master`? Do you want to? Anything else that needs to do be done?

File: docs/source/main/building-security-tools-on-mythx.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (dmuhs) [reply docs/source/main/building-security-tools-on-mythx.rst]: Ah, got it. Try `text` as a lexer name: http://pygments.org/docs/lexers/

**Comment by dmuhs** [comment]
@bmmpxf I'll make the change in the branch and then merge your PR.
The normal workflow should be:
1. Propose a pull request with your changes, assign a reviewer (on the top in the right side bar)
2. Get at least one approval
3. Merge the changes into `master`
4. Go to our ReadTheDocs account
5. In the projects overview, trigger a build for the docs

Then the changes should pop up on the dev guide website :+1:

**Comment by dmuhs** [comment]
All built and pretty at https://docs.mythx.io/en/latest/index.html :tada:

**Pull request merged**